docs/pages/cloud/faq.mdx

```diff
@@ -60,6 +60,17 @@ $ tsh login --proxy=myinstance.teleport.sh:443
 $ tctl status
 ```
 You must use the enterprise version of `tctl`. 
+
+### Why am I getting `permission denied` errors when using `tctl`?
+
+If you have a local file `/etc/teleport.yaml` on your machine `tctl` will attempt to use the local cluster. Set the environment variable `TELEPORT_CONFIG_FILE=` to `""` so it will not attempt to use that Teleport configuration file.
```

Comment:
```suggestion
If you have a local file `/etc/teleport.yaml` on your machine `tctl` will attempt to use the local cluster. Set the environment variable `TELEPORT_CONFIG_FILE` to `""` so it will not attempt to use that Teleport configuration file.
```